libopenthread/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: clean default
+default:
+
+ftd: dependencies
+	make -C libopenthread-ftd
+
+mtd: dependencies
+	make -C libopenthread-mtd
+
+dependencies:
+	make -C libmbedtls
+	make -C libtcplp
+	make -C platform
+
+subcmd:
+	echo "hi"
```

Comment:
?

Reply:
Artifact from testing that should have been removed.